### ISSUE TITLE
[Bug] Can't open dashboard/jobs, always loading

### ISSUE THREAD

**Issue by piscesvivian** [ray-project/ray#21276]
### Search before asking

- [X] I searched the [issues](https://github.com/ray-project/ray/issues) and found no similar issues.

### Ray Component

Dashboard

### What happened + What you expected to happen

1. install ： `pip install ray[default]`
2. start:  `ray start --head --dashboard-host 0.0.0.0 --include-dashboard true`
3. [ERROR] open dasboard, change to jobs page [new dashboard]
dashboard log
```
1132021-12-28 10:15:08,478  INFO web_log.py:206 -- 10.112.60.15 [28/Dec/2021:02:15:08 +0000] "GET /nodes?view=summary HTTP/1.1" 200 5073 "http://10.3.163.101:8265/" "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36"
1142021-12-28 10:15:12,586 INFO web_log.py:206 -- 10.112.60.15 [28/Dec/2021:02:15:12 +0000] "GET /nodes?view=summary HTTP/1.1" 200 5071 "http://10.3.163.101:8265/" "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36"
1152021-12-28 10:15:13,020 INFO node_head.py:270 -- Received a log for 10.3.163.101 and autoscaler
1162021-12-28 10:15:15,285   INFO web_log.py:206 -- 10.112.60.15 [28/Dec/2021:02:15:15 +0000] "GET /jobs?view=summary HTTP/1.1" 404 172 "http://10.3.163.101:8265/" "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36"
1172021-12-28 10:15:18,041   INFO node_head.py:270 -- Received a log for 10.3.163.101 and autoscaler1182021-12-28 10:15:22,959   INFO node_head.py:270 -- Received a log for 10.3.163.101 and autoscaler
1192021-12-28 10:15:25,062   INFO web_log.py:206 -- 10.112.60.15 [28/Dec/2021:02:15:25 +0000] "GET /static/js/2.3f5a3866.chunk.js.map HTTP/1.1" 404 172 "-" "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36"
1202021-12-28 10:15:25,137   INFO web_log.py:206 -- 10.112.60.15 [28/Dec/2021:02:15:25 +0000] "GET /static/js/11.855db36f.chunk.js.map HTTP/1.1" 200 252 "-" "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36"
```

### Versions / Dependencies

Ray==1.9.1
Python==3.8.5,3.7.6
OS==CentOS 7.8 Server NO GPU

### Reproduction script

conda create -n python==3.8.5
pip install ray[default]
pip install ray[tune]
pip install ray[rllib]

ray start --head --dashboard-host 0.0.0.0 --include-dashboard true

[Open] http://<host>:8265
[Click] JOBS

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

**Comment by pablete** [comment]
Also getting a 404 on "GET /jobs?view=summary HTTP/1.1" 

Ray==1.9.0
Python==3.7.6
OS==Ubuntu 18.04 Server NO GPU

**Comment by igorgad** [comment]
Same here. Is this supposed to show jobs submitted with the Job Submission system?

**Comment by scv119** [comment]
cc @simon-mo do you know if this feature is implemented?

**Comment by simon-mo** [comment]
The job submission system recently updated the API, the dashboard page doesn't quite work with that API yet.

**Comment by iycheng** [comment]
I also notice this issue. Are we planning to fix this?

**Comment by lifesboy** [comment]
I've got same issue:
`8265/jobs?view=summary` 404 not found

Ray==1.9.2
Python==3.7.3
OS==Debian 10 (Buster) 4.19.208-1, Nvidia GPU

**Comment by saggar** [comment]
> The job submission system recently updated the API, the dashboard page doesn't quite work with that API yet.

What's the updated REST API path for Jobs listing or summary?

**Comment by simon-mo** [comment]
https://docs.ray.io/en/latest/ray-job-submission/overview.html#rest-api

**Comment by MissiontoMars** [comment]
Is there any progress on this issue? I meet the same problem. @simon-mo

**Comment by alanwguo** [comment]
Update on this, the jobs tab is fixed in ray 2.0
